### PR TITLE
Fix dev caller location

### PR DESCRIPTION
**Pull request by gogiel**
Resolves https://github.com/dejan/rails_panel/issues/144
I was able to reproduce this by running app in `/app` which is a common Docker setup.
`include?` incorrectly matches `meta_request/app_notifications.rb`

### CODE REVIEW

**Comment by gogiel** [comment]
Hi @dejan. Can you find a moment to review it? Are there any other user are capable of merging requests? @modosc ?

I use RailsPanel every day and would love to contribute more.

Thanks!

**Comment by dejan** [comment]
Hi @gogiel, I've added you as a repo collaborator. Thanks for the help!

File: meta_request/test/unit/meta_request/utils_test.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+describe MetaRequest::Utils do
+  describe ".dev_callsite" do
+    it "returns line parsed with Callsite if rails root is found in multi-line trace" do
```
Comment (dejan) [inline meta_request/test/unit/meta_request/utils_test.rb]: I plan to introduce Rubocop soon and default setting for the string literals is the single quote.

File: meta_request/test/unit/meta_request/utils_test.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      stacktrace = [
+        "/gem/gem_file.rb:1:in `func'`",
+        "#{File.join(MetaRequest.rails_root, "test_file.rb")}:87:in `app_func'",
+        "/gem/gem_file.rb:1:in `func2'`",
+      ]
+      expected_callsite_line = Callsite::Line.new("/Users/jan/gat/rails_panel/meta_request/test_file.rb", 87, "app_func")
```
Comment (dejan) [inline meta_request/test/unit/meta_request/utils_test.rb]: Would this test even work on other machines due to this absolute path?

Comment (gogiel) [reply meta_request/test/unit/meta_request/utils_test.rb]: 🤦‍♂ thanks for catching that obvious bug

**Pull request merged**